freqtrade/exchange/exchange.py

```diff
@@ -2055,6 +2055,42 @@ def _set_leverage(
         except ccxt.BaseError as e:
             raise OperationalException(e) from e
 
+    def get_interest_rate(self) -> float:
+        """
+        Calculate interest rate - necessary for Margin trading.
```

Comment:
Interest rates aren't calculated, they're given by the exchange

Reply:
well then the comment is slightly wrong.
However, my assumption was that we might need to calculate it for dry-run. 
At any rate, we'll HAVE to cache it for backtesting - otherwise we'd go to the exchange for every trade in backtesting - but only get the same rate back every time.

Reply:
> well then the comment is slightly wrong. However, my assumption was that we might need to calculate it for dry-run. At any rate, we'll HAVE to cache it for backtesting - otherwise we'd go to the exchange for every trade in backtesting - but only get the same rate back every time.

I don't believe it would ever be calculated, I think that it's something that's kind of like price, where you either have it or you don't